hibernate-core/src/test/java/org/hibernate/orm/test/idgen/userdefined/SimpleBeanContainer.java

```diff
@@ -19,10 +21,8 @@ public <B> ContainedBean<B> getBean(
 			Class<B> beanType,
 			LifecycleOptions lifecycleOptions,
 			BeanInstanceProducer fallbackProducer) {
-		if ( beanType == SimpleGenerator.class ) {
-			return () -> (B) new SimpleGenerator( new AtomicLong( INITIAL_VALUE ) );
-		}
-		return null;
+		return () -> (B) ( beanType == SimpleGenerator.class ?
```

Comment:
Why are these changes needed?

Reply:
It will fail new tests, I was wrong, It shouldn't return `null`.